<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastWidget</class>
 <widget class="QWidget" name="ForecastWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预报</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ForecastWidget {
    background-color: #f5f7fa;
}

QFrame#hourlyCard, QFrame#dailyCard {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e4e7ed;
}

QLabel#sectionTitle {
    font-size: 18px;
    font-weight: bold;
    color: #303133;
}

QLabel#cityLabel {
    font-size: 16px;
    color: #606266;
}

QScrollArea {
    border: none;
    background: transparent;
}

QScrollArea > QWidget > QWidget {
    background: transparent;
}

QFrame.hourItem {
    background-color: #f5f7fa;
    border-radius: 8px;
    min-width: 70px;
    padding: 8px;
}

QFrame.dailyItem {
    background-color: #f5f7fa;
    border-radius: 8px;
    padding: 12px;
}

QLabel.timeLabel {
    font-size: 12px;
    color: #909399;
}

QLabel.tempLabel {
    font-size: 16px;
    font-weight: bold;
    color: #303133;
}

QLabel.iconLabel {
    font-size: 24px;
}

QLabel.dateLabel {
    font-size: 14px;
    font-weight: bold;
    color: #303133;
}

QLabel.weekdayLabel {
    font-size: 12px;
    color: #909399;
}

QLabel.highTempLabel {
    font-size: 16px;
    font-weight: bold;
    color: #F56C6C;
}

QLabel.lowTempLabel {
    font-size: 16px;
    font-weight: bold;
    color: #409EFF;
}

QPushButton#refreshBtn {
    background-color: #409EFF;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 14px;
}

QPushButton#refreshBtn:hover {
    background-color: #66b1ff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>选择城市查看预报</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="hourlyCard">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="hourlyCardLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="hourlyTitle">
        <property name="text">
         <string>逐小时预报</string>
        </property>
        <property name="styleSheet">
         <string>font-size: 16px; font-weight: bold; color: #303133;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="hourlyScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <widget class="QWidget" name="hourlyScrollContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>760</width>
           <height>120</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hourlyLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="dailyCard">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="dailyCardLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="dailyTitle">
        <property name="text">
         <string>7日预报</string>
        </property>
        <property name="styleSheet">
         <string>font-size: 16px; font-weight: bold; color: #303133;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="dailyScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="dailyScrollContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>760</width>
           <height>280</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dailyLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
